environments/swiss/public.yml

```diff
@@ -46,12 +46,14 @@ blobdb_s3: False
 couch_s3: False
 
 
-postgresql_max_connections: 300
-pgbouncer_max_connections: 400
 pgbouncer_default_pool: 290
-pgbouncer_reserve_pool: 5
-pgbouncer_pool_timeout: 1
+pgbouncer_max_connections: 400
 pgbouncer_pool_mode: transaction
+pgbouncer_pool_timeout: 1
+pgbouncer_reserve_pool: 5
+postgresql_max_connections: 300
+postgresql_wal_keep_segments: 0
+postgresql_work_mem: '1MB'
```

Comment:
don't know what the last two do, but assume this is just documenting what's already there?

Reply:
Yeah that's right. The point here was to update the config here to what swiss currently has, so that deploying to swiss doesn't change it in any significant way.

Reply:
i agree with this, we should make it ansible-deployable first, then start changing stuff using ansible.